### PR TITLE
feat(i18n): check no slug duplicates

### PR DESCRIPTION
**Pull request by ojeytonwilliams**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes nothing, but should ensure that if CI passes, we can deploy.

### CODE REVIEW

**Review by naomi-lgbt — APPROVED** [review]
Awesome. This should help catch everything.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome

**Review by Sboonny — APPROVED** [review]
LGTM 🎉, thank you for your contribution to the page! 👍.

File: tools/challenge-auditor/index.ts [hunk 1]
```diff
@@ -26,9 +35,32 @@ const superBlockFolderMap = {
 // the audit, we just ignore them.
 const blocksThatNeedToMove = ['d3-dashboard'];
 
+// Adding types for getChallengesForLang is possible, but not worth the effort
+// at this time.
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
```
Comment (Sboonny) [inline tools/challenge-auditor/index.ts:43]: TearsGIF

Comment (Sboonny) [reply tools/challenge-auditor/index.ts:43]: I can actually post gifs here🤣

Comment (ojeytonwilliams) [reply tools/challenge-auditor/index.ts:43]: I wouldn't worry about it @Sboonny. We would have to declare the types or migrate getChallenges.js.  The former is a maintenance headache (we'll likely change getChallenges in the not-to-distant future) and the latter is *hard*.

**Pull request merged**